sql/core/benchmarks/ExternalAppendOnlyUnsafeRowArrayBenchmark-results.txt

```diff
@@ -0,0 +1,50 @@
+================================================================================================
+WITHOUT SPILL
+================================================================================================
+
+Java HotSpot(TM) 64-Bit Server VM 1.8.0_162-b12 on Mac OS X 10.13.6
+Intel(R) Core(TM) i7-4870HQ CPU @ 2.50GHz
+
+Array with 100000 rows:                  Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
+------------------------------------------------------------------------------------------------
+ArrayBuffer                                   4810 / 5120         21.3          47.0       1.0X
+ExternalAppendOnlyUnsafeRowArray              4996 / 5043         20.5          48.8       1.0X
+
+Java HotSpot(TM) 64-Bit Server VM 1.8.0_162-b12 on Mac OS X 10.13.6
+Intel(R) Core(TM) i7-4870HQ CPU @ 2.50GHz
+
+Array with 1000 rows:                    Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
+------------------------------------------------------------------------------------------------
+ArrayBuffer                                   8839 / 8951         29.7          33.7       1.0X
+ExternalAppendOnlyUnsafeRowArray              9884 / 9888         26.5          37.7       0.9X
```

Comment:
Could you run this once more in your side? For me, I've got the followings. The ratio difference is too big.

**Mac**
```
[info] Java HotSpot(TM) 64-Bit Server VM 1.8.0_191-b12 on Mac OS X 10.14.2
[info] Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz
[info] Array with 1000 rows:                    Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
[info] ------------------------------------------------------------------------------------------------
[info] ArrayBuffer                                 10226 / 10272         25.6          39.0       1.0X
[info] ExternalAppendOnlyUnsafeRowArray            24301 / 24425         10.8          92.7       0.4X
```

**EC2 Server**
```
[info] OpenJDK 64-Bit Server VM 1.8.0_191-b12 on Linux 3.10.0-862.3.2.el7.x86_64
[info] Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz
[info] Array with 1000 rows:                    Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
[info] ------------------------------------------------------------------------------------------------
[info] ArrayBuffer                                 11988 / 12027         21.9          45.7       1.0X
[info] ExternalAppendOnlyUnsafeRowArray            37480 / 37574          7.0         143.0       0.3X
```

Reply:
The is the only difference.

Reply:
I'm rerunning it soon

Reply:
Got the same ratio as you this time:
```
[info] Java HotSpot(TM) 64-Bit Server VM 1.8.0_162-b12 on Mac OS X 10.13.6
[info] Intel(R) Core(TM) i7-4870HQ CPU @ 2.50GHz
[info] Array with 1000 rows:                    Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
[info] ------------------------------------------------------------------------------------------------
[info] ArrayBuffer                                 10028 / 10197         26.1          38.3       1.0X
[info] ExternalAppendOnlyUnsafeRowArray            30053 / 30312          8.7         114.6       0.3X
```

Reply:
Thank you for confirmation. Master branch seems to be changed.